Modules/_cursesmodule.c

```diff
@@ -3255,6 +3255,7 @@ _curses_setupterm_impl(PyObject *module, const char *term, int fd)
     Py_RETURN_NONE;
 }
 
+#if (!defined(_AIX) || (defined(NCURSES_VERSION))
```

Comment:
There are redundant parenthesis.
```suggestion
#if !defined(_AIX) || defined(NCURSES_VERSION)
```

Reply:
done